### ISSUE TITLE
I001 isort found an import in the wrong position

### ISSUE THREAD

**Issue by idgserpro** [gforcada/flake8-isort#23]
I have a ```setup.cfg``` file with the following configuration:

```cfg
[isort]
force_alphabetical_sort = True
force_single_line = True
lines_after_imports = 2
line_length = 200
not_skip = __init__.py
```
I have a file with the following imports:

```python
from plone.app.testing import applyProfile
from plone.app.testing import FunctionalTesting
```
The import order is correct, but flake8-isort 2.1.2 indicates the error:

```
I001 isort found an import in the wrong position
```

**Comment by gforcada** [comment]
@idgserpro if you run isort it does not complain right? On the meantime you can stick with 2.0.x releases until we fix it in 2.1.3.

@cas-- does it ring a bell?

**Comment by cas--** [comment]
It's along the same lines as wrapped imports, it's rather annoying that isort in_lines doesn't match out_lines, so it strictly is an isort bug but it's easy to modify the existing wrapped lines fix:

On [line 176](https://github.com/gforcada/flake8-isort/blob/master/flake8_isort.py#L176) removing the space in `if '\n ' in line:` so it becomes `if '\n' in line:` should solve this.

I didn't expect a wrapped line without indenting so I shall switch to using `splitlines()` instead...

**Comment by idgserpro** [comment]
@gforcada  when I run isort:
```bash
isort testing.py
```
it leaves in the correct order.

**Comment by gforcada** [comment]
@idgserpro would you mind giving a try to 2.1.3? I just released it with #23 on it.

**Comment by hvelarde** [comment]
is fixed, thanks!

**Comment by jrbenny35** [comment]
Hello, I am getting this error running isort 2.2.1, python 3.6.0, and tox 2.7.0. I am running it within tox as well as with the default config.

The error indicates the first 2 lines are in the wrong position or I001:
```
from selenium.common.exceptions import NoSuchElementException
from selenium.webdriver.common.by import By

from foxpuppet import expected
from foxpuppet.windows import BaseWindow
from foxpuppet.windows.browser.navbar import NavBar
from foxpuppet.windows.browser.notifications import BaseNotification
```
And it also says that there is an ```I004 isort found an unexpected blank line in imports``` for the blank line inbetween on line 3. However running isort directly on the file shows no errors. Any tips?

**Comment by cas--** [comment]
@jrbenny35 Please create a new issue, this one is closed!
